release_notes/unreleased.md

```diff
@@ -1 +1,3 @@
 **Unreleased**
+* Fixed the automation triggering issue for 'on poll' [PAPP-24628]
+* Fixed a bug for the 'list offenses' action [PAPP-23735]
```

Comment:
suggestion: Would it be possible to be more specific about the bug that gets fixed for this action?

Reply:
Updated the release notes